data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			`	Grayden, Braiden and Kaiden all start shaking each others' hands and congratulating each other on being part of something so inspirational. After their celebration is over, Kaiden turns back to you. "Our transport fleet is at your disposal. They are being refitted at Mani in the Avior system."`
+			`	The three stand up in unison. Grayden and Braiden leave, and Kaiden turns from the door. "We are so happy you stopped by today and gave us this opportunity to really give back to the needy communities out in our neighborhood." With that, they are all gone, and your escort returns to lead you out.`
+
+
+
+mission "Home for Skadenga 5"
+	name "Transport for Skadenga"
+	landing
+	description "Travel to <destination> to prepare the Transport Union Fleet."
+	to offer
+		has	"asgard"
+	source "Valhalla"
+	destination "Mani"
+	on complete
+		event "mani refit" 21
+		conversation
+			apply
+				set "Asgard2"
+			scene "scene/citydark"
+			`Mani is a quiet, empty moon with only a few hubs of human activity. The Deep Secruity forces, Transport Union, and other semi-governmental fleets are all serviced here.`
+			`	A Transport Union executive meets you when you touch down. "It'll take a while for the ships to be ready, Captain <last>. We're primitive-proofing them so that those ice-savages from Nifel don't accidently kill themselves. I'll send you a message when the ships are ready. It should take about three weeks," the executive explains as he skims through a time table displayed on his tablet.` 
+			`	"When the ships return covered in hay, feces, and other low-tech trash, it'll take us weeks to get them in a usable state again," he mutters before promptly turning and leaving.`
+
+
+
+event "mani refit"
+
+
+
+mission "Mani Refit"
+	name "Mani Refit"
+	to offer
+		has "event: mani refit"
+		has "Home for Skadenga 5: done"
+	description "Head back to <destination> and meet up with the Transport Union's Fleet."
+	source
+		government "Republic"
+	destination "Mani"
+	on offer
+		dialog
+			`Your inbox blinks with a new message from the Transport Union.`
+			`	'The fleet is ready for its inspirational journey. Please return to Mani.`
+			`		-Jayden, Mani Administrator, Transport Union`
+
+
+
+mission "Home for Skadenga 6"
+	landing
+	name "Transport the Skadenga"
+	description `Escort a fleet of freighters, refitted for carrying primitives, to <destination>.`
+	source "Mani"
+	destination "Nifel"
+	to offer
+		has	"Asgard2"
+	on offer
+		conversation
+			`You arrive at the sapceport to see a Union Executive overseeing the final preparations of his fleet. After a moment, the last crewmembers leave his side and board their ships. He walks over to you. "The fleet is ready to launch. Please, try to bring them back to me in one piece."`
+			choice
+				`	"Let's get this over with."`
```

Comment:
Some extra dialogue options here would be appreciated. Doesn't have to lead to different responses or anything.

Reply:
Added a few more options.